wifi-subsys/main/main.c

```diff
@@ -22,19 +22,45 @@
  *
  */
 #include <stdio.h>
+#include <string.h>
 #include "sdkconfig.h"
 #include "freertos/FreeRTOS.h"
 #include "freertos/task.h"
 #include "esp_system.h"
+#include <esp_log.h>
```

Comment:
it's not a std library
```suggestion
#include "esp_log.h"
```

Reply:
this still untouched